parcels/codegenerator.py

```diff
@@ -255,7 +255,7 @@ def visit_Name(self, node):
             node = StatusCodeNode(math, ccode='')
         elif node.id == 'math':
             node = MathNode(math, ccode='')
-        elif node.id == 'random':
+        elif node.id == 'ParcelsRandom':
```

Comment:
interesting that this change was required - I can't remember that I needed to do that. Well spotted!